CMakeLists.txt

```
@@ -591,6 +593,10 @@ if(CP2K_USE_LIBXC)
   find_package(LibXC 7 REQUIRED)
 endif()
 
+if(CP2K_USE_HDF5)
+  find_package(HDF5 REQUIRED COMPONENTS C Fortran)
```

```suggestion
  find_package(HDF5 REQUIRED CONFIG COMPONENTS C Fortran)
```
Worth making the `CONFIG` requirement explicit?

---

Actually, this does't work. Better to use the CMake-provided one (i.e. without `CONFIG`).

---

I use the cmake one although there are inconsistencies with the targets naming. But that ok